frontend/templates/troubleshooting/index.tsx

```diff
@@ -262,16 +189,85 @@ const Wiki: NextPageWithLayout<{
                      {wikiData.linkedProblems.length > 0 && (
                         <RelatedProblems problems={wikiData.linkedProblems} />
                      )}
-                     <PixelPing id={id} type="wiki" />
-                  </Flex>
-               </Container>
-            </Flex>
+                  </Stack>
+               </Stack>
+            </Box>
          </TOCContextProvider>
          {viewStats && <ViewStats {...viewStats} />}
+         <PixelPing id={id} type="wiki" />
       </>
    );
 };
 
+function Heading({ wikiData }: { wikiData: TroubleshootingData }) {
+   const { title, mainImageUrl, mainImageUrlLarge } = wikiData;
+   const { isOpen, onOpen, onClose } = useDisclosure();
+   const lastUpdatedDate = new Date(wikiData.lastUpdatedDate * 1000);
+   const smBreakpoint = useToken('breakpoints', 'sm');
+   const imageSx: any = {
+      display: 'none',
+   };
+   imageSx[`@media (min-width: ${smBreakpoint})`] = {
+      display: 'block',
+   };
```

Comment:
I believe this is the valid syntax to combine these lines:

```suggestion
   const imageSx: any = {
      display: 'none',
      [`@media (min-width: ${smBreakpoint})`]: {
         display: 'block'
      },
   };
```